examples/website/src/shared/route/articleTemplateContainer.tsx

```diff
@@ -129,20 +129,21 @@ export function ArticleTemplateContainer({id, slug}: ArticleTemplateContainerPro
     socialMediaDescription,
     socialMediaImage,
     socialMediaAuthors,
-    comments
+    comments,
+    canonicalUrl
   } = articleData
 
   const path = ArticleRoute.reverse({id, slug})
-  const canonicalURL = canonicalHost + path
```

Comment:
I'd keep `canonicalURL` and add a second const called `canonicalPeerURL`

Reply:
@roosnic1 check the updates please